appengine/cloudsql/README.md

```diff
@@ -6,7 +6,10 @@ Before you can run or deploy the sample, you will need to create a [Cloud SQL in
 ## Running locally
 1. You will need to be running a local instance of MySQL.
 1. Update the connection string in ``appengine-web.xml`` with your local MySQL instance values.
-    $ mvn clean appengine:devserver
+    
+    `$ mvn clean appengine:devserver`
```

Comment:
Might want to put in a code block instead of code formatting it.

`mvn clean ...`

versus

```
mvn clean ...
```